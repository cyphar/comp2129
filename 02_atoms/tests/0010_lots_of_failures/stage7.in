LOAD nonexist
LOAD bad file name
LOAD
START 2 3 2
LOAD shouldnotwork
LOAD bad file name
LOAD
LOAD nonexist
LOAD a.a
