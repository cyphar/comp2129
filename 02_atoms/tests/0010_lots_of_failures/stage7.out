Cannot Load Save

Too Many Arguments

Missing Argument

Game Ready
Red's Turn

Restart Application To Load Save

Restart Application To Load Save

Restart Application To Load Save

Restart Application To Load Save

Restart Application To Load Save

